storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -7375,6 +7589,10 @@ int main(int argc, char **argv)
 	}
 #endif
 
+	if (opt_check_privileges) {
```

Comment:
Good point. Maybe xb_init then? Right after connect and before reading server variables?